connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/aiagent/memory/conversation/document/CamundaDocumentConversationSession.java

```diff
@@ -55,14 +55,14 @@ public CamundaDocumentConversationSession(
       CamundaDocumentStore documentStore,
       ObjectMapper objectMapper,
       ObjectWriter objectWriter,
```

Comment:
We could delegate serialization/deserialization to a dedicated class to avoid passing both `objectMapper` and `objectWriter`.

Reply:
Good point! Done in f65e03023476d3fe0250aea73fd6c79cdf0e0c78.